lib/samples/show_grid/show_grid_sample.dart

```diff
@@ -0,0 +1,368 @@
+//
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import 'package:arcgis_maps/arcgis_maps.dart';
+import 'package:flutter/material.dart';
+
+class ShowGridSample extends StatefulWidget {
```

Comment:
Oh you also need to add the new mixin `with SampleStateSupport`. We should probably update the doc.
See:
https://github.com/Esri/arcgis-maps-sdk-flutter-samples/pull/47